### PR TITLE
page with posts of subscribed categories

### PR DESCRIPTION
**Pull request by valfutz**

### CODE REVIEW

**Review by NykoEugen — COMMENTED** [review]
Look's good for me. One thing should be done, a button for user subscribtions somwhere.

File: subscription/urls.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from subscription.views import PostsOfSubscribedCategoriesView
+
+app_name = "subscription"
+
+urlpatterns = [
+    path(
+        "posts-of-subcribed/",
```
Comment (acman) [inline subscription/urls.py:9]: lets it be only `posts`, it will be displayed as `subscription/posts`

File: subscription/views.py [hunk 2]
```diff
@@ -22,3 +28,32 @@ def update_subscriptions(self, selected_categories: Any, user: Any) -> None:
             user_subscriptions = Subscription.objects.filter(user=self.user)
 
         user_subscriptions.first().categories.set(selected_categories)
+
+
+class PostsOfSubscribedCategoriesView(LoginRequiredMixin, View):
```
Comment (acman) [inline subscription/views.py:33]:
```suggestion
class SubscriptionPostView(LoginRequiredMixin, View):
```

**Review by acman — CHANGES_REQUESTED** [review]
Nice job! Rename please all parts to something more clear like SubscriptionPost, subscription_post_list.html and so on

**Review by acman — APPROVED** [review]
👍

**Pull request merged**